rllib/algorithms/algorithm.py

```diff
@@ -680,6 +681,7 @@ def setup(self, config: PartialAlgorithmConfigDict):
                 mod, obj = method_type.rsplit(".", 1)
                 mod = importlib.import_module(mod)
                 method_type = getattr(mod, obj)
+
             if isinstance(method_type, type) and issubclass(
```

Comment:
`OffPolicyEstimator` is a subclass of `OfflineEvaluator`. So I think the code between these two conditions can be shared. You basically need to instantiate the classes in the end but the only difference is to add `model_config['gamma'] = self.config["gamma"]` only if it's also a subclass of `OffPolicyEstimator`.

Reply:
ok, updated. 👍